daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -721,6 +783,7 @@ func (devices *DeviceSet) setupBaseImage() error {
 		return err
 	}
 
+	devices.saveBaseDeviceUUID(info)
```

Comment:
is the error returned by `saveBaseDeviceUUID` ignored here on purpose? should we return it?